cmd/server.go

```diff
@@ -15,6 +15,7 @@ package cmd
 
 import (
 	"context"
+	"github.com/pingcap/ticdc/pkg/logutil"
```

Comment:
BTW, could you move the import to line 29?

Reply:
get!